### PR TITLE
Issue #365 teleport entity placement

### LINKED ISSUE #365: Teleport entity placement
Drawing teleports should be easier for user. A way to achieve it could be adding "outer vertex" instead of "triangle base" beneath the cursor. This will provide sufficient distance (triangle height) from an obstruction.

Somehow linked to #366 - both issues are related with navmesh margins at the end of the day.

### PR DESCRIPTION
**Pull request by AleksanderKamienski**
I changed teleports drawing rule in apainter. When i was testing the change, i noticed bug connected with rset logic located in worker.py (thats why there is also one line deletion in worker.py).

Belowe is buidling geometry using which I found a bug in worker.py

[cad.json](https://github.com/user-attachments/files/17815140/cad.json)

### CODE REVIEW

File: evac/worker.py
Comment (kowalskiw) [inline evac/worker.py]: Potential issue with RSET will be explored in issue #369. Please, restore those changes.

**Review by kowalskiw — CHANGES_REQUESTED** [review]
RSET to be left as it was previously. The rest LGTM

**Comment by AleksanderKamienski** [comment]
I restored rset in worker.py. I will wait for the approval of PR  #367 and then I will resolve the conflicts here because maybe there will be some changes in the code and more conflicts will appear

**Pull request closed without merging**
